packages/bolt-connection/test/channel/node/node-channel.test.js

```diff
@@ -59,13 +59,13 @@ describe('NodeChannel', () => {
   })
 
   describe('.setupReceiveTimeout()', () => {
-    it('should call socket.setTimeout(receiveTimeout)', () => {
+    it('should not call socket.setTimeout(receiveTimeout)', () => {
```

Comment:
nit: given the test name, I'd expect to see an assertion about the fact socket.setTimeout being called 0 times in the test, but I don't see any

Reply:
It always called when the connection is established